tests/test_dte_data_models.py

```diff
@@ -222,70 +298,270 @@ def test_constants_match(self):
             DteDataL2.DATETIME_FIELDS_TZ,
         )
 
-    def test_init_fail(self) -> None:
-        # TODO: implement for 'DteDataL2()'
-        pass
+    def test_ok_razon_social_none(self) -> None:
+        _ = dataclasses.replace(
+            self.dte_l2_1,
+            emisor_razon_social=None,
+            receptor_razon_social=None,
+        )
```

Comment:
See comment for `DteDataL1Test.test_is_ok_negative_monto_total_in_tipo_dte_liquidacion_factura`.